packages/sdk/src/lib/types/coreTypes.ts

```diff
@@ -14,7 +14,8 @@ export type DataScalarType =
   | bigint
   | string
   | Uint8Array
-  | ArrayBuffer;
+  | ArrayBuffer
+  | File;
```

Comment:
Accept File, don't ask builder to encode it themselves into ArrayBuffer or Uint8Array.